CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [6.1.2] - 2023-04-14
+
+### Changed
+
+- s3 bucket ownership control permission and object ownership to custom resource for CloudFront logging bucket
+- changed xml2js version to 0.5.0
+
```

Comment:
Not a big deal but I thought you mentioned adding a link to the S3 default behavior changes so there would be more context into why the change was made.

Reply:
I have a link to the announcement in the description as a starting point The announcement for the Amazon S3 change can be found at https://aws.amazon.com/about-aws/whats-new/2022/12/amazon-s3-automatically-enable-block-public-access-disable-access-control-lists-buckets-april-2023/